src/apps/appstore/views/marketWelcome.html

```diff
@@ -1,3 +1,6 @@
+<div class="market-beta-notice">
+	<i class="fa fa-info-circle"></i> {{ i18n.marketplace.betaNotice }}
+</div>
```

Comment:
I think it would be better to use the [monsterPanelText](https://docs.2600hz.com/ui/docs/handlebars/#monsterpaneltext) handlebars helper that is available in Monster UI, which allows to display informational messages using the UI look and feel.